### PR TITLE
[Reviewer AMC] Switch ENT logging channel to local7

### PR DESCRIPTION
**Pull request by steelyvoid**
Switch to a less congested logging facility for ENT logs

### CODE REVIEW

**Comment by steelyvoid** [comment]
Hi Andy.  Can you rereview this and the sprout pull to check that the refactorisation is what you were expecting?  I'll assign both PRs to you

File: src/pdlog.cpp [hunk 1]
```diff
@@ -0,0 +1,55 @@
+/**
+ * @file pdlog.cpp
+ *
+ * Project Clearwater - IMS in the Cloud
+ * Copyright (C) 2013  Metaswitch Networks Ltd
```
Comment (bossmc) [inline src/pdlog.cpp]: Year

File: src/pdlog.cpp [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+#include <boost/filesystem.hpp>
+
+void PDLogStatic::init(char *pname)
+{
+  // Get the full path of the executable from the first command line argument
+  boost::filesystem::path p = pname;
+
+  // Copy the filename to a string so that we can be sure of its lifespan -
+  // the value passed to openlog must be valid for the duration of the program.
+  std::string *filename = new std::string(p.filename().c_str());
```
Comment (bossmc) [inline src/pdlog.cpp]: You leak this... but maybe we don't care.

File: include/pdlog.h [hunk 3]
```diff
@@ -50,32 +50,38 @@ extern "C" {
 #include "syslog_facade.h"
 }
 
+// Namespace for common static ENT logging functions
+namespace PDLogStatic
```
Comment (bossmc) [inline include/pdlog.h]: It's weird to call these by two names.  Either the comment should say PDLog (and our docs should too) or this namespace should be called ENTLogStatic (or just ENTLog tbh).

**Comment by bossmc** [comment]
Couple of comments, looks generally good.

File: include/pdlog.h (hunk 3 above)
Comment (steelyvoid) [inline include/pdlog.h]: Leaving as is following phone discussion

**Pull request merged**